operator/redisfailover/service/generator.go

```diff
@@ -47,7 +47,10 @@ sentinel parallel-syncs mymaster 2`
 	graceTime = 30
 )
 
-const redisHAProxyName = "redis-haproxy"
+const (
+	redisHAProxyName     = "redis-haproxy"
+	redisHAProxyHostName = "redis-haproxy"
```

Comment:
Curious, is there a reason we need to have 2 constants of the same string?

Reply:
absolutely agreed